app/src/main/res/layout/fragment_home.xml

```diff
@@ -69,14 +69,40 @@
         android:layout_height="wrap_content"
         app:layout_behavior="com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior">
 
-        <androidx.recyclerview.widget.RecyclerView
```

Comment:
the fragment is still called `home`, is it ok?

Reply:
yes it will be renamed after Davran complete his work
not to make 100500 conflicts